README.md

```diff
@@ -1,8 +1,10 @@
 [![License: MPL 2.0](https://img.shields.io/badge/License-MPL%202.0-brightgreen.svg)](https://opensource.org/licenses/MPL-2.0)
+[![Build](https://travis-ci.org/mozilla-services/channelserver.svg?branch=master)](https://travis-ci.org/mozilla-services/channelserver)
 
-# pairsona
+# ChannelServer
 
-Pairing work for communication between devices/applications.
+A project to make Firefox Account log-in, pairing and sync easier
+between devices and applications devices or applications.
```

Comment:
nit: "devices and applications devices or applications"?

Reply:
gah, thanks!